news/_posts/2021-03-02-archived-modules.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Focusing on essentials modules"
+subtitle: "Get some details about the archived modules"
+date:   2021-03-02 14:00:00
+authors:  [ MarionFrancois ]
+icon: icon-puzzle
+tags: 
+ - modules
+ - news
+---
+
+The product team recently did a survey of 600 PrestaShop users to find out which modules were essential and which they used the least.
```

Comment:
Exactly 600? Otherwise you can say "over 600" or "almost 600"

Reply:
Yes, almost 600 sounds great